Resources/doc/index.rst

```diff
@@ -167,7 +167,7 @@ Method b) A menu as a service
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 For information on how to register a service and tag it as a menu, read
-`Creating Menus as Services <menu_service>`_.
+`Creating Menus as Services <menu_service.rst>`_.
```

Comment:
Actually, this is fixed properly in https://github.com/KnpLabs/KnpMenuBundle/pull/255